### PR TITLE
Setup graphics-shapes publishing

### PR DESCRIPTION
**Pull request by MatkovIvan**
Fixes https://youtrack.jetbrains.com/issue/CMP-5689, https://youtrack.jetbrains.com/issue/CMP-6868

### CODE REVIEW

File: mpp/build.gradle.kts [hunk 1]
```diff
@@ -96,6 +96,12 @@ val libraryToComponents = mapOf(
         ComposeComponent(":compose:material3:adaptive:adaptive-layout"),
         ComposeComponent(":compose:material3:adaptive:adaptive-navigation"),
     ),
+    "GRAPHICS_SHAPES" to listOf(
+        ComposeComponent(
+                path = ":graphics:graphics-shapes",
```
Comment (ASalavei) [inline mpp/build.gradle.kts]:
```suggestion
            path = ":graphics:graphics-shapes",
```

File: graphics/graphics-shapes/gradle.properties [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+artifactRedirecting.publication.targetNames=android,desktop
+artifactRedirecting.androidx.groupId=androidx.graphics
```
Comment (eymar) [inline graphics/graphics-shapes/gradle.properties:18]: there are ios and other targets here https://maven.google.com/web/index.html?q=graphics#androidx.graphics:graphics-shapes-iosarm64 

but for 1.0.1 version.
Can't we use 1.0.1 yet?

File: gradle.properties [hunk 3]
```diff
@@ -124,6 +125,7 @@ artifactRedirecting.androidx.window.version=1.4.0-alpha04
 artifactRedirecting.modules-for-knative-manifest=\
   :annotation:annotation=artifactRedirecting.androidx.annotation.version,\
   :collection:collection=artifactRedirecting.androidx.collection.version,\
+  :graphics:graphics-shapes=artifactRedirecting.androidx.graphics.version,\
```
Comment (eymar) [inline gradle.properties:128]: Probably CI fails due to this line:
> No matching variant of androidx.graphics:graphics-shapes:1.0.0 was found.


afair, this is needed only if we actually redirect to androidx-published libs.
For version 1.0.0 there are no k/native artefacts on googleMaven. 
But if we can use 1.0.1, then this line is needed.

File: graphics/graphics-shapes/gradle.properties (hunk 2 above)
Comment (MatkovIvan) [reply graphics/graphics-shapes/gradle.properties:18]: Done

File: gradle.properties (hunk 3 above)
Comment (MatkovIvan) [reply gradle.properties:128]: Updated to 1.0.1

**Review by eymar — APPROVED** [review]

**Pull request merged**